pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 homepage = "https://github.com/harrymander/dataclasses-struct"
 repository = "https://github.com/harrymander/dataclasses-struct"
 documentation = "https://github.com/harrymander/dataclasses-struct/blob/main/README.md#usage"
-packages = [{include = "dataclasses_struct"}]
+packages = [{ include = "dataclasses_struct" }, { include = "dataclasses_struct/py.typed" }]
```

Comment:
This is not needed since Poetry automatically includes everything in the package source dir. (You can check by installing in a virtualenv and the `py.typed` file should be in the `lib/site-packages/dataclasses_struct` dir)

Reply:
Right. Was basing this on a few examples that specifically add it in setup.py as it isn't a normal source file, but I guess Poetry knows about it and included it already.